docs/documentation/release_notes/topics/26_0_0.adoc

```diff
@@ -104,6 +104,10 @@ when exporting a realm.
 
 For information on how to migrate, see the link:{upgradingguide_link}[{upgradingguide_name}].
 
+= Keycloak Organization is now a supported feature
+
+The organizations feature, introduced in {project_name} 25 as technology preview, is now fully supported.
+
```

Comment:
Not sure we should use {project_name} here because we don't have a RHBK 25, do we?

Reply:
true, so I'll probably revert back, release notes are pulled into RHBK docs and we do not want to have `Keycloak` there, right?